### ISSUE TITLE
Native Widnows control

### ISSUE THREAD

**Issue by OfekIshay** [rivafarabi/deckboard#375]
I was wondering if it's possible to make this app work without the phone, so you could use it native on Windows, I have a second small touch screen connected to my PC and I can only use the deckboard with an android emulator.
is it possible to make it native?

Thanks!

**Comment by kineticscreen** [comment]
Yes this! Be able to switch it between 'edit' and 'operate' or something similar. Also a useful way to test configurations without having to go between the phone and PC.

**Comment by s7726** [comment]
As an electron app this is essentially a web app already... I'd like to see the app host a webapp client. Then you can open that on another monitor, win/max/linx tablet, etc.

**Comment by lordgreggreg** [comment]
This would be perfect for extra touch screens to use

**Comment by OfekIshay** [comment]
> This would be perfect for extra touch screens to use

That is exactly my intended purpose!
